packages/realtime_client/lib/src/realtime_client.dart

```diff
@@ -89,6 +90,8 @@ class RealtimeClient {
   };
   int longpollerTimeout = 20000;
   SocketStates? connState;
+  // This is called `accessToken` in realtime-js
+  Future<String> Function()? customAccessToken;
```

Comment:
This gets nowhere set, right? I guess this should be added to the constructor?

Reply:
Oops, great call!